CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## 0.12.17-wip
+## 0.12.17
```

Comment:
@natebosch thoughts on doing `0.12.16+1` to avoid a package:test release?

Reply:
SGTM.